Tests/L0/VsTest/_suite.ts

```diff
@@ -116,320 +117,322 @@ describe('VsTest Suite', function() {
             psr.run(path.join(__dirname, 'DefaultTestResultsDirectoryIsUsedIfOverrideParamsAreUsed.ps1'), done);
         })
     }
+    if (os.type().match(/^Win/)) {
```

Comment:
or better if the task throws error for non-win platform and the one test case validates that